tests/clv/test_utils.py

```diff
@@ -948,6 +948,7 @@ def test_expected_cumulative_transactions_date_index(fitted_bg, cdnow_trans):
     observation_period_end = "19970930"
     time_scaler = 1
 
+    # Still tracking this function down
```

Comment:
The equivalent `pymc-marketing` function is `rfm_summary`.

Reply:
Thanks. 

I'm getting a bit confused now because the result doesn't have dates in it, right? Feel like this function is doing a lot

Reply:
There is a `set_index_date` argument. If `True` then the returned dataframe will have a datetime index. By default it's `False` and the index can be interpreted as the number of time periods since the origin, but I'm not entirely sure how useful of a feature this is.